src/utils/countdown.spec.js

```diff
@@ -0,0 +1,44 @@
+import fakeTimers from '@sinonjs/fake-timers';
+import Countdown from './countdown';
+
+describe('Countdown', () => {
+  let clock;
+
+  beforeEach(() => {
+    clock = fakeTimers.install(window);
+  });
+  afterEach(() => {
+    if (clock) {
+      clock.uninstall();
+    }
+  });
+  it('should finish the countdown', () => {
+    const countdown = new Countdown();
+    countdown.start(20);
+    clock.tick(20000);
+
+    expect(countdown.secondsLeft).toBe(0);
```

Comment:
Could we check also that the should clear interval?

Reply:
I tested what the stop function is called, I was trying to test the clear interval but I couldn't.

Tell me is that's enough.

Reply:
You can check the number of timers left to be executed:

```
expect(clock.countTimers()).toBe(0);
```